src/Core/Models/Request/TokenRequest.cs

```diff
@@ -19,9 +19,10 @@ public class TokenRequest
         public bool? Remember { get; set; }
         public string CaptchaToken { get; set; }
         public DeviceRequest Device { get; set; }
+        public string OrgIdentifier { get; set; }
```

Comment:
I think the changes in this file are based off some old commits on jslib, which I ended up reverting. You should be able to revert the changes in this file.

Reply:
84c25e0760ba3fedc0e235ca48a330253f96ef81